Dockerfile

```diff
@@ -1,5 +1,5 @@
 ############# builder
-FROM golang:1.23.0 AS builder
+FROM golang:1.23 AS builder
```

Comment:
Removing the patch version from the tests is great and it will make dependabot updates work out of the box for all specific go minor versions.

For the main images I think it's best to be very explicit